EnglishTranslation-master/QUEST.tsv

```diff
@@ -3610,246 +3610,246 @@ QUEST_20160224_003617	I don't think I'll be able to ask for any more help today.
 QUEST_20160310_003734	Medicine on top organized from brightest to darkest color.{nl}Red medicine on the left contains dangerous ingredients.{nl}Ingredients at the bottom organized from heaviest to lightest.{nl}Report missing ingredients on the corresponding sheet.{nl}For other questions, contact Sarma.{nl}{S30} - Sarma, Head of Research -{/}{nl}
 QUEST_20160310_003735	Before you read this, please make sure no one else is around.{nl}I'm Gatre and I'm helping Sarma with his experiments.{nl}Sarma approached me saying he wanted to do research to get the forest's fruit trees back to normal.{nl}
 QUEST_20160310_003736	We had lost a place to live after the trees grew too big and we couldn't eat the fruits, so Sarma's proposal appealed to us.{nl}He offered to share food with whoever helped him so most people from my village ended up accepting.{nl}
 QUEST_20160310_003737	At first we worked hard to help him save our hometown.{nl}But then Sarma's research started to feel somewhat strange.{nl}It seems like Sarma's research has little to do with getting the overgrown trees back to normal.{nl}The experiments are mostly done on monsters, not on trees, and I've even fallen unconscious during one experiment.{nl}
-QUEST_20160310_003738	I suspect Sarma is doing this to cover up for something evil.{nl}But because I have no proof, there's really nothing I can do.{nl}I know I can't ask you to trust my every word.{nl}But at least I want to know if my suspicions are correct, or if I'm simply overthinking it.{nl}
+QUEST_20160310_003738	I suspect Sarma is doing this to cover up for something evil.{nl}But because I have no proof, there's really nothing I can do.{nl}I know I can't ask you to trust every single word of mine.{nl}But at least I want to know if my suspicions are correct, or if I'm simply overthinking it.{nl}
 QUEST_20160310_003739	I'm being watched by Sarma almost constantly. {nl}Please understand that this is the only way I could communicate this to you.{nl}If you agree to help me, when you bring back this manual please say you did everything according to it.
 QUEST_20160310_003740	Discarded Document
-QUEST_20160310_003741	For the experiment, we collected juice and seeds from the fruits of the overgrown trees.{nl}We were lost in despair, but now we feel hope again.{nl}I only wish this research will bring back our town��
-QUEST_20160310_003742	Sarma asked us to capture a weak monster.{nl}We aren't fit for combat, but we did our best, for the village, and managed to catch a monster.{nl}A few people were injured in the process.
+QUEST_20160310_003741	For the experiment, we collected juice and seeds from the fruits of the overgrown trees.{nl}We were lost in despair, but now we feel hope again.{nl}I only wish this research will bring back our town...
+QUEST_20160310_003742	Sarma asked us to capture a weak monster.{nl}We aren't fit for fighting, but we did our best, for the village, and managed to catch a monster.{nl}A few people were injured in the process.
```

Comment:
`We aren't fit for fighting, but we did our best for the village and managed to catch a monster.{nl}Only a few people were injured in the process.`